Using MAPPING_STR=22221111, VERSION_DIR=v4
Running /platform/source/s01/v4/m22221111/m22221111_q10_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 5: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 8: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 11: wget: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 11: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 12: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 13: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 14: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 17: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Collecting git+https://github.com/packagename/direct_redis.git
  Cloning https://github.com/packagename/direct_redis.git to /tmp/pip-req-build-fhj7o2dz
  Running command git clone -q https://github.com/packagename/direct_redis.git /tmp/pip-req-build-fhj7o2dz
[31m  ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git clone -q https://github.com/packagename/direct_redis.git /tmp/pip-req-build-fhj7o2dz[0m
[31mERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?[0m
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 29: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 30: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 31: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q10_t0.sh: line 32: sudo: command not found
Running /platform/source/s01/v4/m22221111/m22221111_q11_t0.sh
Running /platform/source/s01/v4/m22221111/m22221111_q12_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q12_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q12_t0.sh: line 7: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Running /platform/source/s01/v4/m22221111/m22221111_q13_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q13_t0.sh: line 5: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q13_t0.sh: line 8: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Running /platform/source/s01/v4/m22221111/m22221111_q14_t0.sh
Collecting pip
  Downloading pip-23.3.1-py3-none-any.whl (2.1 MB)
[?25l[K     |‚ñè                               | 10 kB 8.6 MB/s eta 0:00:01[K     |‚ñé                               | 20 kB 4.8 MB/s eta 0:00:01[K     |‚ñå                               | 30 kB 7.1 MB/s eta 0:00:01[K     |‚ñã                               | 40 kB 4.4 MB/s eta 0:00:01[K     |‚ñä                               | 51 kB 4.8 MB/s eta 0:00:01[K     |‚ñà                               | 61 kB 5.7 MB/s eta 0:00:01[K     |‚ñà                               | 71 kB 6.5 MB/s eta 0:00:01[K     |‚ñà‚ñé                              | 81 kB 6.3 MB/s eta 0:00:01[K     |‚ñà‚ñç                              | 92 kB 7.0 MB/s eta 0:00:01[K     |‚ñà‚ñå                              | 102 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñä                              | 112 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñâ                              | 122 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà                              | 133 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñè                             | 143 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñé                             | 153 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñå                             | 163 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñã                             | 174 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñâ                             | 184 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà                             | 194 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà                             | 204 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñé                            | 215 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñç                            | 225 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñã                            | 235 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñä                            | 245 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñâ                            | 256 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà                            | 266 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñè                           | 276 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñç                           | 286 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñå                           | 296 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñã                           | 307 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñâ                           | 317 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà                           | 327 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                          | 337 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                          | 348 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                          | 358 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                          | 368 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                          | 378 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                          | 389 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                          | 399 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                         | 409 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                         | 419 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                         | 430 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                         | 440 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                         | 450 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                         | 460 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                        | 471 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                        | 481 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                        | 491 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                        | 501 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                        | 512 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                        | 522 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                        | 532 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                       | 542 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                       | 552 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                       | 563 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                       | 573 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                       | 583 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                       | 593 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                      | 604 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                      | 614 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                      | 624 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                      | 634 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                      | 645 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                      | 655 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                      | 665 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                     | 675 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                     | 686 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                     | 696 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                     | 706 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                     | 716 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                     | 727 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                    | 737 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                    | 747 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                    | 757 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                    | 768 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                    | 778 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                    | 788 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                   | 798 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                   | 808 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                   | 819 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                   | 829 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                   | 839 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                   | 849 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                   | 860 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                  | 870 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                  | 880 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                  | 890 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                  | 901 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                  | 911 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                  | 921 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                 | 931 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                 | 942 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                 | 952 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                 | 962 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                 | 972 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                 | 983 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                 | 993 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé                | 1.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                | 1.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå                | 1.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä                | 1.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                | 1.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                | 1.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè               | 1.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé               | 1.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå               | 1.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã               | 1.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ               | 1.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà               | 1.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà               | 1.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé              | 1.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç              | 1.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã              | 1.2 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä              | 1.2 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ              | 1.2 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà              | 1.2 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè             | 1.2 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç             | 1.2 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå             | 1.2 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã             | 1.2 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ             | 1.2 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà             | 1.2 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè            | 1.3 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé            | 1.3 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç            | 1.3 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã            | 1.3 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä            | 1.3 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà            | 1.3 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà            | 1.3 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè           | 1.3 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç           | 1.3 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå           | 1.4 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä           | 1.4 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ           | 1.4 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà           | 1.4 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè          | 1.4 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé          | 1.4 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå          | 1.4 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã          | 1.4 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä          | 1.4 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà          | 1.4 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà          | 1.5 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé         | 1.5 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç         | 1.5 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå         | 1.5 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä         | 1.5 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ         | 1.5 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà         | 1.5 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè        | 1.5 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé        | 1.5 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå        | 1.5 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã        | 1.6 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ        | 1.6 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà        | 1.6 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà        | 1.6 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé       | 1.6 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç       | 1.6 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã       | 1.6 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä       | 1.6 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ       | 1.6 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà       | 1.6 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè      | 1.7 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç      | 1.7 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå      | 1.7 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã      | 1.7 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ      | 1.7 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà      | 1.7 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè     | 1.7 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé     | 1.7 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç     | 1.7 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã     | 1.8 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä     | 1.8 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà     | 1.8 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà     | 1.8 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè    | 1.8 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç    | 1.8 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå    | 1.8 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä    | 1.8 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ    | 1.8 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà    | 1.8 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè   | 1.9 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé   | 1.9 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå   | 1.9 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã   | 1.9 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä   | 1.9 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà   | 1.9 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà   | 1.9 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé  | 1.9 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç  | 1.9 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå  | 1.9 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä  | 2.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ  | 2.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà  | 2.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè | 2.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé | 2.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå | 2.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã | 2.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ | 2.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà | 2.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà | 2.0 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñé| 2.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç| 2.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã| 2.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñä| 2.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ| 2.1 MB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 2.1 MB 6.7 MB/s 
[?25hInstalling collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 20.0.2
    Not uninstalling pip at /usr/lib/python3/dist-packages, outside environment /usr
    Can't uninstall 'pip'. No files were found to uninstall.
Successfully installed pip-23.3.1
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4/m22221111/m22221111_q15_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Get:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease [114 kB]
0% [2 InRelease 11.5 kB/114 kB 10%]                                   0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
0% [Waiting for headers]0% [Waiting for headers]                        Get:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease [114 kB]
0% [4 InRelease 5583 B/114 kB 5%]                                 0% [Working]20% [Waiting for headers]                         Get:5 http://ports.ubuntu.com/ubuntu-ports focal-updates/main arm64 Packages [2804 kB]
20% [5 Packages 8477 B/2804 kB 0%]                                  49% [Working]             Get:6 http://ports.ubuntu.com/ubuntu-ports focal-updates/universe arm64 Packages [1329 kB]
50% [6 Packages 93.9 kB/1329 kB 7%]50% [5 Packages store 0 B] [6 Packages 93.9 kB/1329 kB 7%]                                                          63% [5 Packages store 0 B] [Waiting for headers]                                                63% [Waiting for headers]63% [6 Packages store 0 B] [Waiting for headers]                                                Get:7 http://ports.ubuntu.com/ubuntu-ports focal-security/universe arm64 Packages [1023 kB]
63% [6 Packages store 0 B] [7 Packages 0 B/1023 kB 0%]                                                      Get:8 http://ports.ubuntu.com/ubuntu-ports focal-security/main arm64 Packages [2408 kB]
74% [6 Packages store 0 B] [8 Packages 20.5 kB/2408 kB 1%]                                                          92% [8 Packages 1713 kB/2408 kB 71%]92% [7 Packages store 0 B] [8 Packages 1746 kB/2408 kB 72%]                                                           99% [7 Packages store 0 B]                          99% [Working]99% [8 Packages store 0 B]                          100% [Working]              Fetched 7791 kB in 2s (3653 kB/s)
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
python3 set to manually installed.
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q16_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q16_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q16_t0.sh: line 7: sudo: command not found
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q17_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mCollecting git+https://github.com/your-repository/direct-redis.git
  Cloning https://github.com/your-repository/direct-redis.git to /tmp/pip-req-build-hldpjkki
[31m  ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git version[0m[31m
[0m[31mERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?[0m[31m
[0mRequirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q18_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q18_t0.sh: line 4: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q19_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q19_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q19_t0.sh: line 7: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q1_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q1_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q1_t0.sh: line 7: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q20_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q20_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q20_t0.sh: line 7: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0m[31mERROR: Could not find a version that satisfies the requirement direct_redis_package (from versions: none)[0m[31m
[0m[31mERROR: No matching distribution found for direct_redis_package[0m[31m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q21_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-dev is already the newest version (3.8.2-0ubuntu2).
python3-dev set to manually installed.
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
Requirement already satisfied: pip in /usr/local/lib/python3.8/dist-packages (23.3.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mThe virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt install python3.8-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.

Failing command: ['/query_env/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']

/platform/source/s01/v4/m22221111/m22221111_q21_t0.sh: line 9: query_env/bin/activate: No such file or directory
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0m/platform/source/s01/v4/m22221111/m22221111_q21_t0.sh: line 15: deactivate: command not found
Running /platform/source/s01/v4/m22221111/m22221111_q22_t0.sh
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q2_t0.sh
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q3_t0.sh
Installing Python dependencies...
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q4_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q4_t0.sh: line 5: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q4_t0.sh: line 8: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q5_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q6_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mpython3: can't open file 'revenue_query.py': [Errno 2] No such file or directory
Running /platform/source/s01/v4/m22221111/m22221111_q7_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q7_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q7_t0.sh: line 7: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0m[33mDEPRECATION: git+https://github.com/20c/direct_redis.git#egg=direct_redis[pandas] contains an egg fragment with a non-PEP 508 name pip 25.0 will enforce this behaviour change. A possible replacement is to use the req @ url syntax, and remove the egg fragment. Discussion can be found at https://github.com/pypa/pip/issues/11617[0m[33m
[0mCollecting direct_redis[pandas]
  Cloning https://github.com/20c/direct_redis.git to /tmp/pip-install-cul9kzu0/direct-redis_6c6735d2893f431ba5401b2af4a74849
[31m  ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git version[0m[31m
[0m[31mERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?[0m[31m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q8_t0.sh
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4/m22221111/m22221111_q9_t0.sh
/platform/source/s01/v4/m22221111/m22221111_q9_t0.sh: line 7: sudo: command not found
/platform/source/s01/v4/m22221111/m22221111_q9_t0.sh: line 10: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mCollecting git+https://github.com/RedisLabsModules/redis-py.git
  Cloning https://github.com/RedisLabsModules/redis-py.git to /tmp/pip-req-build-wqqh_yu1
[31m  ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git version[0m[31m
[0m[31mERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?[0m[31m
[0mRequirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0m/platform/source/s01/v4/m22221111/m22221111_q9_t0.sh: line 22: sudo: command not found
Collecting msgpack
  Downloading msgpack-1.0.7-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (9.1 kB)
Downloading msgpack-1.0.7-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (525 kB)
[?25l   [90m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m [32m0.0/525.4 kB[0m [31m?[0m eta [36m-:--:--[0m[2K   [91m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m[90m‚ï∫[0m[90m‚îÅ‚îÅ[0m [32m491.5/525.4 kB[0m [31m14.8 MB/s[0m eta [36m0:00:01[0m[2K   [90m‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ[0m [32m525.4/525.4 kB[0m [31m13.0 MB/s[0m eta [36m0:00:00[0m
[?25hInstalling collected packages: msgpack
Successfully installed msgpack-1.0.7
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mchmod: cannot access 'query.py': No such file or directory
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q10_t0.py /platform/output/s01/v4/m22221111/m22221111_q10_t0.csv /platform/expected/s01/q10.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q10_t0.py", line 18, in <module>
    nation_df = pd.read_json(nation_data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q11_t0.py /platform/output/s01/v4/m22221111/m22221111_q11_t0.csv /platform/expected/s01/q11.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q11_t0.py", line 24, in <module>
    nation_df, supplier_df = get_nation_supplier_from_redis()
  File "/platform/source/s01/v4/m22221111/m22221111_q11_t0.py", line 18, in get_nation_supplier_from_redis
    nation_df = pd.read_json(rd.get('nation'), orient='records')
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q12_t0.py /platform/output/s01/v4/m22221111/m22221111_q12_t0.csv /platform/expected/s01/q12.csv
>>> Expected ((2, 3))
   HIGH_LINE_COUNT  LOW_LINE_COUNT L_SHIPMODE
0              647             945       MAIL
1              620             943       SHIP

>>> Actual ((0, 3))
Empty DataFrame
Columns: [HIGH_LINE_COUNT, LOW_LINE_COUNT, L_SHIPMODE]
Index: []

Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q13_t0.py /platform/output/s01/v4/m22221111/m22221111_q13_t0.csv /platform/expected/s01/q13.csv
>>> Expected ((37, 2))
    CUSTDIST  C_COUNT
0       5000        0
1        676       10
2        651        9
3        618       11
4        554       12
5        548        8
6        514       13
7        487        7
8        485       19
9        461       18
10       454       14
11       444       20
12       442       16
13       438       17
14       430       15
15       396       21
16       378       22
17       355        6
18       322       23
19       262       24
20       188       25
21       184        5
22       162       26
23       138       27
24       103       28
25        92        4
26        59       29
27        49        3
28        29       30
29        26       31
30        19       32
31        12        2
32         8       33
33         7       34
34         5       35
35         3        1
36         1       36

>>> Actual ((36, 2))
    CUSTDIST  C_COUNT
0       5003        1
1        676       10
2        651        9
3        618       11
4        554       12
5        548        8
6        514       13
7        487        7
8        485       19
9        461       18
10       454       14
11       444       20
12       442       16
13       438       17
14       430       15
15       396       21
16       378       22
17       355        6
18       322       23
19       262       24
20       188       25
21       184        5
22       162       26
23       138       27
24       103       28
25        92        4
26        59       29
27        49        3
28        29       30
29        26       31
30        19       32
31        12        2
32         8       33
33         7       34
34         5       35
35         1       36

Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q14_t0.py /platform/output/s01/v4/m22221111/m22221111_q14_t0.csv /platform/expected/s01/q14.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q14_t0.py", line 26, in <module>
    part_data = pd.read_json(part_data_raw.decode('utf-8'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'decode'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q15_t0.py /platform/output/s01/v4/m22221111/m22221111_q15_t0.csv /platform/expected/s01/q15.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q15_t0.py", line 27, in <module>
    lineitem_df['TOTAL_REVENUE'] = lineitem_df['L_EXTENDEDPRICE'] * (1 - lineitem_df['L_DISCOUNT'])
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 3761, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/indexes/range.py", line 349, in get_loc
    raise KeyError(key)
KeyError: 'L_EXTENDEDPRICE'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q16_t0.py /platform/output/s01/v4/m22221111/m22221111_q16_t0.csv /platform/expected/s01/q16.csv
>>> Expected ((2762, 4))
       P_BRAND  P_SIZE                  P_TYPE  SUPPLIER_CNT
0     Brand#14      45   SMALL ANODIZED NICKEL            12
1     Brand#22      19   SMALL BURNISHED BRASS            12
2     Brand#25      14   PROMO POLISHED COPPER            12
3     Brand#35      45    LARGE ANODIZED STEEL            12
4     Brand#35       9    PROMO BRUSHED COPPER            12
...        ...     ...                     ...           ...
2757  Brand#22      19     PROMO BRUSHED BRASS             3
2758  Brand#22      14  PROMO BURNISHED COPPER             3
2759  Brand#43      23  STANDARD BRUSHED BRASS             3
2760  Brand#44       9  MEDIUM ANODIZED NICKEL             3
2761  Brand#53      49  MEDIUM BURNISHED BRASS             3

[2762 rows x 4 columns]

>>> Actual ((2762, 4))
       P_BRAND  P_SIZE                    P_TYPE  SUPPLIER_CNT
0     Brand#14      45     SMALL ANODIZED NICKEL            12
1     Brand#22      19     SMALL BURNISHED BRASS            12
2     Brand#25      14     PROMO POLISHED COPPER            12
3     Brand#35      45      LARGE ANODIZED STEEL            12
4     Brand#35       9      PROMO BRUSHED COPPER            12
...        ...     ...                       ...           ...
2757  Brand#55      19  STANDARD POLISHED COPPER             4
2758  Brand#55      36  STANDARD POLISHED COPPER             4
2759  Brand#55      14  STANDARD POLISHED NICKEL             4
2760  Brand#55       9   STANDARD POLISHED STEEL             4
2761  Brand#55      36   STANDARD POLISHED STEEL             4

[2762 rows x 4 columns]

Head row similarity: 0.9725283501038173
Score.success_approx
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q17_t0.py /platform/output/s01/v4/m22221111/m22221111_q17_t0.csv /platform/expected/s01/q17.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q17_t0.py", line 20, in <module>
    part_data = pd.read_json(part_data_json, lines=True)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q18_t0.py /platform/output/s01/v4/m22221111/m22221111_q18_t0.csv /platform/expected/s01/q18.csv
Score.fail_timeout
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q19_t0.py /platform/output/s01/v4/m22221111/m22221111_q19_t0.csv /platform/expected/s01/q19.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q19_t0.py", line 17, in <module>
    part_df = pd.read_json(part_data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q1_t0.py /platform/output/s01/v4/m22221111/m22221111_q1_t0.csv /platform/expected/s01/q1.csv
>>> Expected ((4, 10))
   AVG_DISC     AVG_PRICE    AVG_QTY  COUNT_ORDER L_LINESTATUS L_RETURNFLAG  SUM_BASE_PRICE    SUM_CHARGE  SUM_DISC_PRICE  SUM_QTY
0  0.050145  36002.123829  25.537587       147790            F            A    5.320754e+09  5.256751e+09    5.054096e+09  3774200
1  0.049394  35521.326916  25.300664         3765            F            N    1.337378e+08  1.322863e+08    1.271324e+08    95257
2  0.050096  36000.924688  25.545538       292000            O            N    1.051227e+10  1.038558e+10    9.986238e+09  7459297
3  0.049989  35994.029214  25.525944       148301            F            R    5.337951e+09  5.274406e+09    5.071819e+09  3785523

>>> Actual ((0, 10))
Empty DataFrame
Columns: [AVG_DISC, AVG_PRICE, AVG_QTY, COUNT_ORDER, L_LINESTATUS, L_RETURNFLAG, SUM_BASE_PRICE, SUM_CHARGE, SUM_DISC_PRICE, SUM_QTY]
Index: []

Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q20_t0.py /platform/output/s01/v4/m22221111/m22221111_q20_t0.csv /platform/expected/s01/q20.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q20_t0.py", line 17, in <module>
    nation_df = pd.read_json(redis_client.get('nation').decode('utf-8'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'decode'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q21_t0.py /platform/output/s01/v4/m22221111/m22221111_q21_t0.csv /platform/expected/s01/q21.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q21_t0.py", line 60, in <module>
    main()
  File "/platform/source/s01/v4/m22221111/m22221111_q21_t0.py", line 54, in main
    nation, supplier = query_redis(redis_db)
  File "/platform/source/s01/v4/m22221111/m22221111_q21_t0.py", line 19, in query_redis
    nation = DataFrame(eval(redis_db.get('nation')))
TypeError: eval() arg 1 must be a string, bytes or code object
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q22_t0.py /platform/output/s01/v4/m22221111/m22221111_q22_t0.csv /platform/expected/s01/q22.csv
>>> Expected ((4, 3))
   CNTRYCODE  NUMCUST  TOTACCTBAL
0         20       91   666158.10
1         21      101   760953.55
2         22      106   759804.81
3         30       87   646748.02

>>> Actual ((4, 3))
   CNTRYCODE  NUMCUST  TOTACCTBAL
0         20       91   666158.10
1         21      101   760953.55
2         22      107   764789.13
3         30       88   651732.41

Head row similarity: 0.9230769230769231
Score.success_approx
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q2_t0.py /platform/output/s01/v4/m22221111/m22221111_q2_t0.csv /platform/expected/s01/q2.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q2_t0.py", line 15, in <module>
    nation_df = pd.read_msgpack(r.get('nation'))
AttributeError: module 'pandas' has no attribute 'read_msgpack'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q3_t0.py /platform/output/s01/v4/m22221111/m22221111_q3_t0.csv /platform/expected/s01/q3.csv
Score.fail_timeout
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q4_t0.py /platform/output/s01/v4/m22221111/m22221111_q4_t0.csv /platform/expected/s01/q4.csv
Score.fail_timeout
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q5_t0.py /platform/output/s01/v4/m22221111/m22221111_q5_t0.csv /platform/expected/s01/q5.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q5_t0.py", line 23, in <module>
    nation_df = pd.read_json(r.get('nation'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q6_t0.py /platform/output/s01/v4/m22221111/m22221111_q6_t0.csv /platform/expected/s01/q6.csv
Revenue calculated and written to query_output.csv: 11803420.25340003
>>> Expected ((1, 1))
        REVENUE
0  7.115407e+06

>>> Actual ((1, 1))
        REVENUE
0  1.180342e+07

Head row similarity: 0.8064516129032258
Score.success_approx
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q7_t0.py /platform/output/s01/v4/m22221111/m22221111_q7_t0.csv /platform/expected/s01/q7.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q7_t0.py", line 19, in <module>
    nation_df = pd.read_json(redis_conn.get('nation'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q8_t0.py /platform/output/s01/v4/m22221111/m22221111_q8_t0.csv /platform/expected/s01/q8.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q8_t0.py", line 24, in <module>
    nation = pd.read_msgpack(r.get('nation'))
AttributeError: module 'pandas' has no attribute 'read_msgpack'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4/m22221111/m22221111_q9_t0.py /platform/output/s01/v4/m22221111/m22221111_q9_t0.csv /platform/expected/s01/q9.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4/m22221111/m22221111_q9_t0.py", line 22, in <module>
    nation = pd.read_msgpack(redis.get('nation'))
AttributeError: module 'pandas' has no attribute 'read_msgpack'
Score.fail_execute
